tests/integration/test_pip_packages.py

```diff
@@ -22,43 +26,47 @@ def test_all_pip_packages(env_name, test_env, tmpdir):
     * Check that expected pip dependencies are identified in response
     * Check response parameters of the package
     * Check that the source tarball includes the application source code
-    * Check that the source tarball includes expected deps/pip directory
+    * Check that the source tarball includes expected deps/pip directory # TODO change name
     * Check: The content manifest is successfully generated and contains correct content
     """
-    env_data = test_env["pip_packages"][env_name]
+    env_data = test_env[env_package][env_name]
     client = utils.Client(test_env["api_url"], test_env["api_auth_type"], test_env.get("timeout"))
-    initial_response = client.create_new_request(
-        payload={
-            "repo": env_data["repo"],
-            "ref": env_data["ref"],
-            "pkg_managers": ["pip"],
-            # TODO: delete pip-dev-preview flag when
-            #  the pip package manager will be ready for production usage
-            "flags": ["pip-dev-preview"],
-        },
-    )
+    if env_package == "pip_packages":
```

Comment:
We can avoid it because we don't need pip_flag anymore.

Reply:
we still need it, https://github.com/release-engineering/cachito/pull/288 was not merged yet